curve25519-dalek/src/edwards.rs

```diff
@@ -2182,7 +2182,7 @@ mod test {
         }
     }
 
-    #[cfg(feature = "alloc")]
+    #[cfg(all(feature = "alloc", feature = "rand_core"))]
     #[test]
     fn compress_batch() {
         let mut rng = rand::rng();
```

Comment:
Why `rng()` here but `OsRng` everywhere else? This is a nit,  doesn’t matter either way